### DISCUSSION TITLE
Artists join value

### DISCUSSION THREAD

**Discussion by GustavAndreasson** [joalla/discogs_client#73] [Support] [answered]
Hi!
I have made an discogs collection viewer application that is currently using my own code to query discogs (https://github.com/GustavAndreasson/records2). There is a lot of room for improvement and I would like to use this library instead. 

One thing that I can't find is the join values of artist lists for realeases with multiple artists (like https://www.discogs.com/release/452087-Rowland-S-Howard-Lydia-Lunch-Shotgun-Wedding ("-" in this case)) so that I can display how the artists are shown on the record. Is there a way to get that information through this library?

**Comment by AnssiAhola** [comment, marked as answer]
Hi, @GustavAndreasson 

There seems to be "join" field in the returned JSON from the release API point
You can get it by first getting artists from the `Release` object and then `fetch` ing the "join" field like so
```python
# Rowland S. Howard - Lydia Lunch – Shotgun Wedding
release_id = 452087
print(client.release(release_id).artists[0].fetch('join'))
# Prints -
```

I'm curious, why does it matter what character you join the artists with? Can't you just join always with "-" 😅 

Anyways, hope that helps

Cheers!
Anssi

**Reply by GustavAndreasson** [reply]
Great! Thank you!

The reason for the join attribute is that I want to show the correct artists names as they are used for the record.

Like in this example, https://www.discogs.com/release/1417827-Captain-Beefheart-And-His-Magic-Band-Safe-As-Milk. Without the join the artist would just be "Captain Beefheart - His Magic Band" while it should be "Captain Beefheart And His Magic Band".
